### DISCUSSION TITLE
GitHub Enterprise and VS Code

### DISCUSSION THREAD

**Discussion by gbrac** [community/community#113688] [Enterprise] [answered]
How do you authenticate and clone a repository using Visual Studio Code with GitHub Enterprise? Everything I can find for steps (and everything in the VS Code tool) is geared towards regular GitHub, not GitHub Enterprise.

**Comment by ettaboyle** [comment, marked as answer]
Hi @gbrac - I asked around and received the following guidance from some GitHub folks:

> - It should be the same way. GitHub Enterprise is GitHub. They browse to the repo they want to clone, click the local button, copy the clone URL, and away they go.

> - When they log in on VS Code, they'll get access to whatever Enterprise orgs/repos the account they logged in as has access to. Same thing as if they logged into [github.com](http://github.com/)

Does that help? Please add additional context if not and I'll see what I can find.

**Reply by bxtp4p** [reply]
Hi @gbrac - Yes, to @ettaboyle's second bullet point, what you have access to in Visual Studio Code depends on who you've logged in as in the IDE. If the account you've logged in as is a member of an enterprise, you should have access to everything you normally have access to, as if you were on github.com. 

For example, this shows me logged in with an account that belongs to an organization that is part of a GitHub Enterprise and I'm able to begin the cloning process of an org repo:

<img width="1377" alt="image" src="https://github.com/community/community/assets/10562371/0b9ae06d-17f8-432b-98b5-b6383a6129f9">
